pages/integrations/inputs/o365_input.rst

```diff
@@ -0,0 +1,54 @@
+.. _o365_input:
+
+***********************
+Office 365 Integrations
+***********************
+
+.. attention:: This is a Graylog Enterprise feature and is only available since Graylog version 3.3. A valid Graylog Enterprise license is required.
+               It is meant for the hosted "online" office and do not use this to collect logs from a windows server, in your data center.
+
+Prerequisites
```

Comment:
i would make this a note

Reply:
@cobaltclaudia -  please feel free to checkout branch o365_documentation and make changes .